src/site/content/en/notifications/use-push-notifications-to-engage-users/index.md

```diff
@@ -0,0 +1,162 @@
+
+The look-and-feel of notifications varies between platforms. For example: 
+
+<figure class="w-figure">
+  <img src="./predicaments-android.png" alt="A notification on an android device.">
+  <figcaption class="w-figcaption">A notification on an android device.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./predicaments-macbook.png" alt="A notification on a MacBook.">
+  <figcaption class="w-figcaption">A notification on a MacBook.</figcaption>
+</figure>
+
+Traditionally, web browsers had to initiate the exchange of information between server and client by making a request. [Web push technology](https://developer.mozilla.org/en-US/docs/Web/API/Push_API), on the other hand, lets you configure your server to send notifications when it makes sense for your app. A push service creates unique URLs for each subscribed service worker. Sending messages to a service worker's URL raises events on that service worker, prompting it to display a notification.      
+
+Push notifications can help users to get the most out of your app by prompting them to re-open it and use it based on the latest information. 
+
+## Creating and sending notifications
+
+Create notifications using the [Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API). A `Notification` object has a `title` string and an `options` object. For example:
+
+```js
+let title = 'Hi!';
+let options = { 
+  body: 'Very Important Message',
+  /* other options can go here */
+};
+let notification = new Notification(title, options);
+```
+
+The `title` is displayed in bold when the notification is active, while the `body` contains additional text.
+
+### Get permission to send notifications
+
+To display a notification, your app must get permission from the user to do so: 
+
+```js
+Notification.requestPermission();
+```
+
+{% Aside %}
+
+The web platform encourages developers to request notifications permission when users understand what is being asked and why. 
+
+Bring up the permission prompt **in response to a user gesture** and **when the value of notifications is obvious**. Avoid bringing up the permission prompt immediately when the user lands on the page.
+
+{% endAside %}
+
+## How do push notifications work?
+
+The real power of notifications comes from the combination of service workers and push technology:
+
+*   [Service workers](https://developers.google.com/web/fundamentals/primers/service-workers) can run in the background and display notifications even when your app isn't visible on screen. 
+
+*   Push technology lets you configure your server to send notifications when it makes sense for your app. A push service creates unique URLs for each subscribed service worker. Sending messages to a service worker's URL raises events on that service worker, prompting it to display a notification.
```

Comment:
"unique URLs" here should probably be subscriptions again. The subscription contains more than just a URL.

https://web-push-book.gauntface.com/chapter-02/01-subscribing-a-user/#what-is-a-pushsubscription